hw/drivers/stm32_rtc/stm32_rtc.h

```diff
@@ -0,0 +1,30 @@
+/* stm32_rtc.h
+ * STM32F4xx RTC implementation ported to STM32F4xx
+ * RebbleOS
+ *
+ * Author: Barry Carter <barry.carter@gmail.com>
+ * Refactored by Thomas England (crc-32) <thomas.c.england@gmail.com>
+ */
+
+// make sure we use the external osc
+#ifndef __STM32_RTC_H
+#define __STM32_RTC_H
+
+#if defined(STM32F4XX)
```

Comment:
I *think* these includes should not be necessary in this file

Reply:
Does not build without, I thought the same